prow/config.yaml

```diff
@@ -446,9 +446,9 @@ tide:
     kubernetes/website: squash
   pr_status_base_url: https://prow.k8s.io/pr
   blocker_label: merge-blocker
-  squash_label: tide/squash
-  rebase_label: tide/rebase
-  merge_label: tide/merge
+  squash_label: tide/merge-method-squash
+  rebase_label: tide/merge-method-rebase
+  merge_label: tide/merge-method-merge
```

Comment:
I think you'll technically want to migrate the label in `label_sync` if you are renaming it -- check if it's in use in any in-flight PRs right now?

Reply:
ah! I missed that was possible. Very cool! updated.